x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/results_view/results_view.tsx

```diff
@@ -67,6 +74,19 @@ export const ResultsView: FC<Props> = ({
           <EuiSpacer size="m" />
 
           <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
```

Comment:
The bottom bar used to have a 'Cancel' button on it which took you back to the entry point of the file data viz. Now the only way to go back and import another file (before importing) is to use the breadcrumbs or hit the browser back button. What do you think about adding a 'Select a different file' type button into here?

Reply:
I've added a button next to the file name title

![image](https://github.com/elastic/kibana/assets/22172091/16372fd8-18d9-4302-a10c-b5156d67a84a)

Reply:
The link you add to 'Select a different file' LGTM

<img width="522" alt="image" src="https://github.com/elastic/kibana/assets/7405507/d7f1fa65-2dd8-48c9-9b7a-99a32480950c">